libs/agent-scripts/cloudbroker/grid/update_node.py

```diff
@@ -0,0 +1,25 @@
+from JumpScale import j
+from urlparse import urlparse
+
+descr = """
+This script updates nodes version
+"""
+
+category = "cloudbroker"
+organization = "greenitglobe"
+license = "bsd"
+version = "1.0"
+roles = []
+async = True
+
+
+def action(nodename):
+    cmd = "sudo kubectl get service | grep management-ssh | awk '{print $3}'"
```

Comment:
kubectl get service management-ssh -o=jsonpath="{.spec.clusterIP}"
can get you the IP